data/json/monsters/zed_lieutenant.json

```diff
@@ -0,0 +1,239 @@
+      {
+        "u_transform_radius": { "global_val": "ltnt_transform_radius", "default": 5 },
+        "ter_furn_transform": "ltnt_grass_drain",
+        "target_var": { "global_val": "drain_terrain" }
+      },
+      {
+        "u_transform_radius": { "global_val": "ltnt_transform_radius", "default": 5 },
+        "ter_furn_transform": "ltnt_tree_drain",
+        "target_var": { "global_val": "drain_terrain" }
+      },
+      {
+        "u_spawn_monster": "GROUP_AMALGAMATIONS_SPAWNED",
+        "group": true,
+        "real_count": 6,
+        "lifespan": [ "30 seconds", "60 seconds" ],
+        "target_var": { "global_val": "drain_terrain" }
+      },
+      { "u_message": "The air hisses!", "type": "warning" }
+    ]
+  },
+  {
+    "type": "ter_furn_transform",
+    "id": "ltnt_grass_drain",
+    "//": "Yes this is a giant list of terrain.",
+    "terrain": [
+      {
+        "result": [ "t_dirt" ],
+        "valid_terrain": [
+          "t_grass",
+          "t_grass_alien",
+          "t_grass_dead",
+          "t_bamboo_long",
+          "t_grass_long",
+          "t_bamboo_tall",
+          "t_grass_tall",
+          "t_bamboo",
+          "t_bamboo_harvested",
+          "t_shrub",
+          "t_shrub_rose",
+          "t_shrub_lilac",
+          "t_shrub_grape",
+          "t_shrub_peanut",
+          "t_shrub_blueberry",
+          "t_shrub_hydrangea",
+          "t_shrub_raspberry",
+          "t_shrub_strawberry",
+          "t_shrub_blackberry",
+          "t_shrub_huckleberry",
+          "t_shrub_rose_harvested",
+          "t_shrub_grape_harvested",
+          "t_shrub_lilac_harvested",
+          "t_shrub_peanut_harvested",
+          "t_shrub_raspberry_harvested",
+          "t_shrub_blueberry_harvested",
+          "t_shrub_hydrangea_harvested",
+          "t_shrub_blackberry_harvested",
+          "t_shrub_strawberry_harvested",
+          "t_shrub_huckleberry_harvested",
+          "t_forestfloor",
+          "t_moss",
+          "t_underbrush",
+          "t_underbrush_harvested_spring",
+          "t_underbrush_harvested_summer",
+          "t_underbrush_harvested_autumn",
+          "t_underbrush_harvested_winter",
+          "t_tree_young"
+        ]
+      }
+    ]
+  },
+  {
+    "type": "ter_furn_transform",
+    "id": "ltnt_tree_drain",
+    "//": "TODO: Add all the tree types! All of them! Including the harvested ones! ;_;",
+    "terrain": [ { "result": [ "t_tree_very_dead" ], "valid_terrain": [ "t_tree" ] } ]
+  },
+  {
+    "name": "GROUP_AMALGAMATIONS_SPAWNED",
+    "type": "monstergroup",
+    "//": "TODO: Borrow more of Venera's fun little twisted creations to explain why biomass = instant abberations.",
+    "monsters": [ { "monster": "mon_amalgamation_swarmer", "weight": 3, "pack_size": [ 4, 6 ] } ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_LIEUTENANT_ACTIVATE_SHADOW",
+    "recurrence": 1,
+    "//1": "TODO: 6 months into the cataclysm AND # of zombie kills?",
```

Comment:
It should already know the exodii, so maybe also track the progression with them? The longer since the Cataclysm and the more exodii activity, the higher the chance to spawn or something. Would be cool at least.

And/or also track number of necromancers killed, those are also relatively high ranking I think.

Reply:
You are suggesting it should appear earlier if the player has met the exodii and/or killed a bunch of necromancers?

Reply:
I am still quite open to changing this condition, but I do want to see how it shakes out. Regarding the exodii, I think that they'd like to hear one of these guys has been around. I considered having dialogue to ask Rubik about it with the (their main computer - I can't recall its name) interrogating you before deciding that 1) They don't have any spare neurolinks to their virtual world to ~~lie-detector~~ properly ask you (how plot fortunate!) and 2) This is bad news, you're fucked, but also good luck against the Great Enemy

But, exodii dialogue is hard to write and I've never actually hooked up dialogue before. This has been a bundle on my plate as-is.

Reply:
Yes and no, these are just ideas and maybe even out of scope here, so feel free to just resolve this conversation if it is enough as it is right now.